Rasa_Bot/domain.yml

```diff
@@ -21,10 +21,38 @@ slots:
     influence_conversation: true
   
 responses:
+  # Fallback
+  utter_default:
+  - text: "Sorry I couldn't understand that (yet). Can you rephrase?"
+
+  # General
   utter_greet:
   - text: "Hey {name}!"
   - text: "Hi {name}!"
   - text: "Hallo {name}!"
+  - text: "Hoi {name}!"
+  utter_goodbye:
+  - text: "Doei!"
+  - text: "Tot straks!"
+  - text: "Tot ziens!"
+  utter_ask_back:
+  - text: "En jij?"
+  utter_agent_name:
+  - text: "Ik ben Kris. Ik ben je virtuele coach."
```

Comment:
Did we settle on this name? Maybe we should turn this into a constant (not sure how that's done within rasa) so we can always easily change the name?

Reply:
And does it refer to Kristell? 😜